demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -341,7 +345,7 @@ def copy_packs_content_to_old_bundles(self, packs):
                 if dir_name == 'TestPlaybooks':
                     self.copy_test_files(sub_dir_path)
                 elif dir_name == RELEASE_NOTES_DIR:
-                    continue
+                    self.copy_test_files(sub_dir_path)
```

Comment:
why is this needed?

Reply:
No need, forgot to remove.
Removed this now.